alerts/openshift-virtualization-operator/NodeNetworkInterfaceDown.md

```diff
@@ -54,15 +54,9 @@ Network interface failures can lead to:
    - Test the interface with a different cable/port
 
 2. For software or configuration issues:
-   ```bash
-   # Restart NetworkManager
-   systemctl restart NetworkManager
-   ```
 
-   ```bash
-   # Bring interface up manually
-   ip link set <interface-name> up
-   ```
+   [OCPV:NodeNetworkInterfaceDown alert for unused network
```

Comment:
Did you mean to push this as part of this PR as well? Or should this be separate?